src/main/java/net/kaczmarzyk/spring/data/jpa/web/WebRequestProcessingContext.java

```diff
@@ -33,6 +35,7 @@
  * It is a wrapper around low-level Spring classes, which provides easier access to e.g. path variables.
  * 
  * @author Tomasz Kaczmarzyk
+ * @author Robert Dworak
```

Comment:
since the development time is paid and managed by Tratif, please update it to:

@author Robert Dworak, Tratif sp. z o.o.